src/main/java/com/google/sps/data/Cache.java

```diff
@@ -11,23 +11,30 @@
 public class Cache {
   private HashMap<Integer,HashMap<Integer,GridCell>> dataGrid = new HashMap<>();
   private int aqDataPointsPerDegree;
+  private static final int NUM_DATA_SOURCES = 2;
 
   public GriddedData getGrid(Coordinates swCorner, Coordinates neCorner) {
     ArrayList<AQDataPoint> data = new ArrayList<>();
     aqDataPointsPerDegree = 100;
-
-    // TODO (Rachel) implement a better failsafe for when both data fetches fail (i.e. a way of checking so the old grid can be returned)
+    int numDataFetchesFailed = 0;
+    
     // Catch the error here because the servlet cannot throw the Exception type
     try {
       data.addAll(getNSWGovData());
     } catch (Exception e) {
       System.out.println(e.getMessage());
+      numDataFetchesFailed ++;
```

Comment:
but if all the other code in the code base has the style of not having a space before ++, doesnt it make sense to apply that consistency here as well?

Reply:
I couldn't find another example (outside of a for loop) that used this. I'll have another look to double check though

Reply:
None of the java files have it in this case.

Reply:
but shouldnt it be consistent overall regardless then?

but it is  a nit

Reply:
Ok, I couldn't find any examples in the examples from the starter projects either, but I had it without spaces in that code, so I'll take the spaces out if for nothing else but to try and get myself to write them consistently

Reply:
I couldn't find anything concrete explicitly called out in the style guide; however, I believe it's "commonly accepted" that there should be no space between the variable and post-increment operator.